atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -50,10 +50,17 @@
   { @selector(selectPreviousTab:), "selectprevioustab" },
   { @selector(mergeAllWindows:), "mergeallwindows" },
   { @selector(moveTabToNewWindow:), "movetabtonewwindow" },
+  { @selector(clearRecentDocuments:), "clearrecentdocuments" },
 };
 
 }  // namespace
 
+// Menu item is located for ease of removing it from the parent owner
+static NSMenuItem* recentDocumentsMenuItem_ = nil;
+
+// Submenu retained to be swapped back to |recentDocumentsMenuItem_|
+static NSMenu* recentDocumentsMenuSwap_ = nil;
```

Comment:
Should I maintain those ownerships with `base::scoped_nsobject`?

Reply:
Found some iOS style guides 😕
https://cs.chromium.org/chromium/src/docs/ios/style.md?q=scoped_nsobject&sq=package:chromium&dr=C&l=8